interface/forms/eye_mag/php/eye_mag_functions.php

```diff
@@ -2104,12 +2107,36 @@ function display_PMSFH($rows, $view = "pending", $min_height = "min-height:344px
                 </tr>
                 </table>
         ';
-
         if (count($PMSFH[0][$key]) > '0') {
             $index=0;
             foreach ($PMSFH[0][$key] as $item) {
-                if (($key == "Medication") && ($item['status'] == "Inactive")) {
-                    continue; }
+                if (
+                        ( ($key == "Medication") || ($key == "Eye Meds") )
+                    &&
+                        ($item['status'] == "Inactive")
+                ) {
+                    continue;
+                }
+                if ( ($key == "Medication") && ( !empty($item['row_subtype'])) ) {
+                    $subtype_Meds[$item['row_subtype']]['name'] = $item['row_subtype'];
+                    $subtype_Meds[$item['row_subtype']]['header'] = '
+                        <table class="PMSFH_header">
+                            <tr>
+                                <td width="90%">
+                                    <span class="left" style="font-weight:800;font-size:0.9em;">'.xlt(ucwords($item['row_subtype'])).' Meds</span>
+                                </td>
+                                <td>
+                                    <span class="right btn-sm" href="#PMH_anchor" onclick="alter_issue2(\'0\',\''.attr($key).'\',\'0\');" style="text-align:right;font-size:8px;">'. xlt("New") .'</span>
```

Comment:
We have a crazy escaping function for when js is embedded in html like the onclick. Can do the following here:
```
<span class="right btn-sm" href="#PMH_anchor" onclick="alter_issue2(\'0\','.attr_js($key).',\'0\');" style="text-align:right;font-size:8px;">'. xlt("New") .'</span>
```
(note used attr_js function and removed the surrounding quotes)

Reply:
need to remove the surrounding quotes on this attr_js call:
```
<span class="right btn-sm" href="#PMH_anchor" onclick="alter_issue2(\'0\','.attr_js($key).',\'0\');" style="text-align:right;font-size:8px;">'. xlt("New") .'</span>
```